models/openvino-conversion-requirements.txt

```diff
@@ -0,0 +1,2 @@
+openvino-dev[pytorch,onnx]
```

Comment:
we can use `openvino>=2023.1.0` which contains update version of `convert_model` directly in main `openvino` pip package, while `openvino-dev` is actually deprecated.